src/AsmResolver.DotNet/Cloning/MemberCloner.cs

```diff
@@ -30,18 +31,29 @@ public partial class MemberCloner
         /// Creates a new instance of the <see cref="MemberCloner"/> class.
         /// </summary>
         /// <param name="targetModule">The target module to copy the members into.</param>
-        public MemberCloner(ModuleDefinition targetModule) : this(targetModule, null) { }
+        /// <param name="callback">The callback used in the cloner listener.</param>
+        public MemberCloner(ModuleDefinition targetModule, Action<IMetadataMember, IMetadataMember>? callback = null) : this(targetModule, null, new CallbackCloneListener(callback ?? new Action<IMetadataMember, IMetadataMember>((orginal, cloned) => { }))) { }
```

Comment:
Please break up long lines :)

Reply:
Also, adding the optional parameter is a breaking ABI change, which means it has to wait till 5.0. If we want this to get into 4.x still, we should maybe split up this constructor into two overloaded instead.